### ISSUE TITLE
DeOldify-API Methods/Commands

### ISSUE THREAD

**Issue by doughnet** [jantic/DeOldify#207]
Howdy.  There are only 2 examples provided but there is no documentation on what other API calls can be done.  Especially how do I do process a local source file rather than a source_url/external?

Appreciate proper documentation.

**Calling the API for image processing**
`
curl -X POST "http://MY_SUPER_API_IP:5000/process" -H "accept: image/png" -H "Content-Type: application/json" -d "{\"source_url\":\"http://www.afrikanheritage.com/wp-content/uploads/2015/08/slave-family-P.jpeg\", \"render_factor\":35}" --output colorized_image.png
`

**Calling the API for video processing**
`curl -X POST "http://MY_SUPER_API_IP:5000/process" -H "accept: application/octet-stream" -H "Content-Type: application/json" -d "{\"source_url\":\"https://v.redd.it/d1ku57kvuf421/HLSPlaylist.m3u8\", \"render_factor\":35}" --output colorized_video.mp4`

**Comment by jantic** [comment]
If you're intending to use the curl api via Docker then the answer is simple:  You can't directly use images on your machine.  The documentation is already "proper" in that regard-  It's not designed to do that. Reason:  It keeps things simple for 99% of users/use cases, and is low maintenance. 

The way around that is to upload the file to a hosting service first, such as Imgur, then use the resulting direct url link to that file.

Now if you really want to go further and do local files then the Jupyter notebooks have examples of that. But those aren't implemented as Docker images, and require more effort/know-how to install (there's instructions for that, too).  

**In short, I'm not here to support everybody's needs/wants. I'm here to put out my research and make it so that others can play around with it and makes something out of it.  That's it.**

**Comment by doughnet** [comment]
thanks.  poor attitude.

**Comment by jantic** [comment]
Sure thing broseph.

**Comment by pnmartinez** [comment]
Adding this feature would be an amazing PR IMO!
